README.md

```diff
@@ -20,19 +20,20 @@ Your entry should maintain alphabetic order and it should be in the format:
 `* your name [username](https://github.com/username)`.
 
 Lines starting with `*` are automatically rendered as bullet points and phrases containing `[name](URL)` are automatically hyperlinked.
-
-* Arun Meena [arun11meena](https://github.com/arun11meena)
-* Avinash Agarwal [manu4rhyme](https://github.com/manu4rhyme)
-* Dushyant Mishra [zero1hac](https://github.com/zero1hac)
-* Govind Sahai [mafiya69](https://github.com/mafiya69)
-* Rehas Sachdeva [rehassachdeva](https://github.com/rehassachdeva)
-* Rishabh Daal [rd13123013](https://github.com/rd13123013)
-* Sartaj Singh [leosartaj](https://github.com/leosartaj)
-* Shashank Shekhar [sskhrnwbie](https://github.com/sshkhrnwbie)
-* Shivam Tyagi [shivamT95](https://github.com/shivamT95)
-* Sumith [Sumith1896](https://github.com/Sumith1896)
-* Yashu Seth [yashu-seth](https://github.com/yashu-seth)
-* Yatna Verma [yatna](https://github.com/yatna)
+After adding the name execute update.py to update the readme.
+
+* Arun Meena [arun11meena](http://github.com/arun11meena)
+* Avinash Agarwal [manu4rhyme](http://github.com/manu4rhyme)
+* Dushyant Mishra [zero1hac](http://github.com/zero1hac)
+* Govind Sahai [mafiya69](http://github.com/mafiya69)
+* Rehas Sachdeva [rehassachdeva](http://github.com/rehassachdeva)
+* Rishabh Daal [rd13123013](http://github.com/rd13123013)
+* Sartaj Singh [leosartaj](http://github.com/leosartaj)
+* Shashank Shekhar [sshkhrnwbie](https://github.com/sshkhrnwbie)
```

Comment:
In both ways the links work perfectly fine. It will be too strict to just restrict to https. Apart from uniformity I don't think this really serves any concrete purpose. Neverthless I will convert them to https.